MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -304,6 +304,17 @@ public bool SupportsFilter(string filter, string option)
             return false;
         }
 
+        public Version GetMediaEncoderVersion()
+        {
+            if (_ffmpegPath != null)
+            {
+                var validator = new EncoderValidator(_logger, _ffmpegPath);
+                return validator.GetFFmpegVersion();
```

Comment:
Can we not save the result somewhere? This method is called every time a transcode is started (and possibly more). It's quite possible that the user has updated or replaced the binary, but the only other place where we validate ffmpeg is during startup and when the ffmpeg path is changed.